File: 206-reverse-linked-list/206-reverse-linked-list.java
class Solution {
    public ListNode reverseList(ListNode head) {
      ListNode prev=null,next=null;
        while(head!=null)
        {
            next=head.next;
            head.next=prev;
            prev=head;
            head=next;
        }
        return prev;
    }
}

File: 206-reverse-linked-list/206-reverse-linked-list.cpp

class Solution {
public:
    ListNode* reverseList(ListNode* head) {
     ListNode *prev=NULL,*next=NULL;
     while(head!=NULL)
     {
         next=head->next;
         head->next=prev;
         prev=head;
         head=next;
         
     }
        return prev;
    }
};